_query-dsl/span/span-multi-term.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Span multi-term
+parent: Span queries
+grand_parent: Query DSL
+nav_order: 40
+---
+
+# Span multi-term query
+
+The `span_multi` query allows you to wrap a multi-term query (like `wildcard`, `fuzzy`, `prefix`, `range`, or `regexp`) as a span query. This enables you to use these more flexible matching queries within other span queries.
+
+For example, you can use the `span_multi` query to:
+- Find words with common prefixes near other terms
+- Match fuzzy variations of words within spans
+- Use regular expressions in span queries
+
+>Using `span_multi` queries can potentially match many terms. To avoid excessive memory usage, you can:
+>- Set the `rewrite` parameter on the multi-term query.
+>- Use the `top_terms_*` rewrite method.
+>- If you use `span_multi` only for a `prefix` query, consider enabling the `index_prefixes` option for the text field instead. This automatically rewrites any `prefix` query on the field into a single-term query that matches the indexed prefix.
```

Comment:
Because this last bullet doesn't complete the stem as the others do, it should be its own sentence (not a bullet) or rewritten to complete the stem.